a3p-integration/README.md

```diff
@@ -33,37 +33,49 @@ make -C ../packages/deployment docker-build-sdk
 
 ## Run the integration test
 
-The `@agoric/synthetic-chain` test runner can be invoked through yarn.
+The `@agoric/synthetic-chain` test runner is invoked through NPM scripts.
 
 To build the agoric-3 synthetic chain images with the unreleased chain software upgrade:
 ```sh
-yarn synthetic-chain build
+yarn build
 ```
 
 To run the unit tests:
 ```sh
-yarn synthetic-chain test
-# or for an interactive session
-yarn synthetic-chain test --debug
+yarn test
+# or for an interactive session into a given proposal layer
+yarn test --debug -m "proposal-name"
 ```
 
 # Proposal structure
 
 ## Layering
 
-Proposal packages are under the `a3p-integration/proposals` folder, and are separate from each other, from the `a3p-integration` test environment, and from the enclosing `agoric-sdk` repository. They run inside the docker image, and cannot access SDK code. The are executed in lexical order.
+Proposal layers are packages under the `a3p-integration/proposals` folder, and are separate from each other, from the `a3p-integration` test environment, and from the enclosing `agoric-sdk` repository. They run inside the docker image, and cannot access SDK code. The are lower case, and executed in lexical order.
+
+In the release branches, the end-to-end `a3p-integration` test usually only has a single proposal layer named `a:upgrade-NN`, which performs a chain software upgrade proposal to the corresponding upgrade plan. In the `master` branch, there may also be core-eval proposal layers, either before or after the chain software upgrade proposal.
+
+The details of a proposal layer are configured through the `agoricProposal` field of its `package.json`.
+
+### Chain software upgrade layer
```

Comment:
This seems like a good place to direct the reader to review https://github.com/Agoric/agoric-3-proposals/blob/main/README.md and https://github.com/Agoric/agoric-3-proposals/tree/main/packages/synthetic-chain. 

Also consider a TODO to move these docs over there and include here only what's specific to agoric-sdk